LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 Jayson Harshbarger
```

Comment:
From dev-branch and readme, it is probably a good idea to choose one or the other.